elyra/pipeline/pipeline.py

```diff
@@ -141,6 +141,14 @@ def env_vars_as_dict(self, logger: Optional[object] = None) -> Dict:
                         print(f"Could not process environment variable entry `{nv}`, skipping...")
         return envs
 
+    @property
+    def image_pull_policy(self):
+        image_tag = self._runtime_image.split(":")[1]
+        if image_tag == "latest":
```

Comment:
Since 'latest' is also implied when a tag does not exist, I believe that should be handled as well.

Reply:
Updated